README.md

```diff
@@ -1 +1,73 @@
-# assignment-CoV2-analysis
+# Analysis Plan
+**By Andrea Chua**
+
+## Analysis 1: using kmer compositions to determine the changes in DNA composition relative to where the virus first began
+
+Virus first began in Wuhan, and data will be collected from NCBI.
+NCBI has the complete genome sequenced here [NCBI](https://www.ncbi.nlm.nih.gov/nuccore/1798174254).
+I am still in the process of finding more information,
+as NCBI doesn't seem to have all the information I need.
+I might need to tweak it to a specific geographical range/region
```

Comment:
You could maybe just search for genomes from China in late 2019 or early 2020. Should be pretty close to ancestral strain